tests/GameTests.cpp

```diff
@@ -24,3 +24,27 @@ TEST_F(GameTests, initial_score_shoud_be_zero)
 {
     ASSERT_EQ(0, game.score());
 }
+
+TEST_F(GameTests, score_should_be_ten_strike)
+{
+    //WHEN
+    game.checkValue("X ");
+    //THEN
+    ASSERT_EQ(10, game.checkValue("X "));
+}
+
+TEST_F(GameTests, score_should_be_ten_spare)
+{
+    //WHEN
+    game.checkValue("2/");
+    //THEN
+    ASSERT_EQ(10, game.checkValue("2/"));
+}
+
+TEST_F(GameTests, score_should_be_nine)
+{
+    //WHEN
+    game.checkValue("45");
+    //THEN
+    ASSERT_EQ(9, game.checkValue("45"));
+}
```

Comment:
Could you add 2 more tests, please - when is miss. 
ASSERT_EQ(7, game.checkValue("-7"));
ASSERT_EQ(3, game.checkValue("3-"));

Reply:
Ok